apps/services/license-api/src/app/modules/license/license.module.ts

```diff
@@ -54,20 +58,11 @@ import { DrivingLicenseApiClientService } from './clients/drivingLicense/driving
       useFactory: (
         disabilityClient: DisabilityLicenseClientService,
         firearmClient: FirearmLicenseApiClientService,
-        drivingLicenseConfig: ConfigType<typeof DrivingLicenseApiClientConfig>,
-        xRoadConfig: ConfigType<typeof XRoadConfig>,
-      ) => async (
-        type: LicenseId,
-        cacheManager: CacheManager,
```

Comment:
FYI the `CacheManager` import is unused now.

Reply:
😩